include/render/VolumeRenderer.h

```diff
@@ -50,6 +50,13 @@ class VolumeRenderer
 	
 		virtual void setClipMinMax(glm::vec3 min_clip, glm::vec3 max_clip) = 0;
 
+
+		void set_animation()
```

Comment:
This function doesn't do anything. Do you need it?

Reply:
removed unused function